doc/introduction/photonic_hardware.rst

```diff
@@ -0,0 +1,95 @@
+Photonic hardware
```

Comment:
Need to look at it in terms of the sidebar, since this will be visible on all pages:
![image](https://user-images.githubusercontent.com/2959003/77131679-6afb6180-6aac-11ea-9118-2422082a248a.png)

Reply:
Still think it could be placed higher - e.g. between Intro and Graphs and Networking. But depends how much we want to highlight at this stage.

Reply:
Yep, I don't really mind where it goes - @co9olguy? Is it something we want to move up when availability is more widespread, or move it up now? Only issue is that currently most users won't be able to provision API keys.

Reply:
Move it on up :)

Reply:
> I was on two fences about the title.. but now I think I like it grin

ouch, *two* fences!?

Reply:
Done!